### DISCUSSION TITLE
在Linux下应该怎么导入7sfunc模板？

### DISCUSSION THREAD

**Discussion by ghost** [hooke007/mpv_PlayKit#515] [Q&A 问答]
[vapoursynth] Script evaluation failed:
[vapoursynth] Python exception: No module named 'k7sfunc'
[vapoursynth] 
[vapoursynth] Traceback (most recent call last):
[vapoursynth]   File "src/cython/vapoursynth.pyx", line 3387, in vapoursynth._vpy_evaluate
[vapoursynth]   File "src/cython/vapoursynth.pyx", line 3388, in vapoursynth._vpy_evaluate
[vapoursynth]   File "/home/_XXXXXX_/.config/mpv/vs/MEMC_MVT_LQ.vpy", line 6, in <module>
[vapoursynth]     import k7sfunc as k7f
[vapoursynth] ModuleNotFoundError: No module named 'k7sfunc'
[vapoursynth] 
[vapoursynth] could not init VS
Disabling filter vapoursynth.00 because it has failed.

我把k7sfunc.py放到MEMC_MVT_LQ.vpy同目录也不行。

**Comment by hooke007** [comment]
查阅Python的linux说明，py脚本在python path下

**Reply by ghost** [reply]
[vapoursynth]     ret = k7f.FPS_CTRL(clip, fps_in=container_fps, fps_ret=True)
[vapoursynth]                        ^^^^
[vapoursynth] NameError: name 'clip' is not defined
现在是这个错误

**Reply by hooke007** [reply]
你脚本怎么写的，没定义clip啊

**Reply by ghost** [reply]
这是自带的啊 我没改动 .config/mpv/vs/MEMC_MVT_LQ.vpy

**Reply by hooke007** [reply]
无法复现，只能猜测是模块函数没导入成功

**Reply by ghost** [reply]
我是把k7sfunc.py放~/.config/mpv下面的.
export PYTHONPATH="~.config/mpv"
然后mpv xxxx.mkv 运行的

**Reply by ghost** [reply]
我的问题，不小心删除了clip = video_in
